oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java

```diff
@@ -107,6 +111,15 @@
         return result;
     }
 
+    private static Map<String, String> buildLabelsWithNodeInfo(Node node,
+                                                               List<LabelKey> keys,
+                                                               List<LabelValue> values) {
+        Map<String, String> result = buildLabels(keys, values);
```

Comment:
Take a look at this comment in the original proto
```
  // This is required only in the first message on the stream or if the
  // previous sent ExportMetricsServiceRequest message has a different Node (e.g.
  // when the same RPC is used to send Metrics from multiple Applications).
  opencensus.proto.agent.common.v1.Node node = 1;
```

You need to hold the previous node, if the current `request#hasNode()` is false, use the previous one.